src/main/java/io/cryostat/net/web/http/api/v1/RecordingsPostHandler.java

```diff
@@ -174,12 +178,18 @@ public void handleAuthenticated(RoutingContext ctx) throws Exception {
                         ? 0
                         : Integer.parseInt(m.group(4).substring(1));
 
+        ConnectionDescriptor connectionDescriptor = getConnectionDescriptorFromContext(ctx);
+        Base32 base32 = new Base32();
```

Comment:
`base32` looks like it could be extracted to a field or constant - doesn't seem like it needs to be instantiated every time.